Gemfile.lock

```diff
@@ -44,23 +45,24 @@ GEM
       ffi (>= 1.15.0)
     eventmachine (1.2.7)
     execjs (2.9.1)
-    faraday (2.9.0)
+    faraday (2.10.1)
       faraday-net_http (>= 2.0, < 3.2)
-    faraday-net_http (3.1.0)
+      logger
+    faraday-net_http (3.1.1)
       net-http
-    ffi (1.16.3)
+    ffi (1.17.0-x86_64-linux-gnu)
```

Comment:
This does not seem right. It's probably going to break users on non Linux machines.